.changes/next-release/bugfix-09d5b2b9-44bc-4305-88ba-a7e54ad87657.json

```
@@ -0,0 +1,4 @@
+{
+  "type" : "bugfix",
+  "description" : "Display spaceName and projectName on the MDE status bar"
```

Users don't know what MDE is

```suggestion
  "type" : "feature",
  "description" : "Display current space and project name on status bar while working in a CodeCatalyst Dev Environment"
```